setup.py

```diff
@@ -60,6 +60,9 @@
                                 'sbol-diff=sbol_utilities.sbol_diff:main']
       },
       packages=['sbol_utilities'],
-      package_data={'sbol_utilities': ['sbolgraph-standalone.js']},
+      package_data={
+          'sbol_utilities': ['sbolgraph-standalone.js'],
+          'sbol_utilities': ['gb2so.csv', 'so2gb.csv']
+      },
```

Comment:
I don't think this will work because there are two entries in the `package_data` dict for `'sbol_utilities'`. These files should all go in one list, and preferably vertically spaced with one file name per line.
```suggestion
      package_data={
          'sbol_utilities': [
              'gb2so.csv',
              'sbolgraph-standalone.js',
              'so2gb.csv',
          ],
      },
```